app/konnect/analytics/use-cases.md

```diff
@@ -1,11 +1,9 @@
 ---
-title: Analyze API Usage and Performance
+title: Custom Reports Use Case - Analyze API Usage and Performance
 content_type: tutorial
-book: use-cases
-chapter: 1
 ---
 
-Custom reporting in {{site.konnect_saas}} gives you the power to monitor your API data in detail and export that data to CSV. 
+Custom reporting in {{site.konnect_saas}} enables you to save, revisit, and share views created during data exploration, facilitating analysis over different time intervals. These reports can be used to analyze metrics, investigate anomalies and sharing. 
```

Comment:
How about we also add that another main use case is that once you saved to a report, you can use this to share and communicate these insights with others?